Solutions/Cybersixgill-Actionable-Alerts/Playbooks/DeleteCybersixgillAlert/azuredeploy.json

```diff
@@ -0,0 +1,437 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "metadata": {
+        "title": "Delete-Cybersixgill-Alert",
+        "description": "This playbook will delete Alert on Cybersixgill portal when resective Incident is deleted in Microsoft Sentinel",
+        "prerequisites": "",

```

Comment:
Enter something in prerequisites that client id and secret key should be stored in key vault before deploying this playbook .